python-sdk/src/astro/sql/__init__.py

```diff
@@ -4,6 +4,8 @@
 
 from astro.sql.operators.append import AppendOperator, append
 from astro.sql.operators.cleanup import CleanupOperator, cleanup
+from astro.sql.operators.data_validations.check_column import ColumnCheckOperator, check_column
```

Comment:
Should we add it `__all__` on L20 ?

Reply:
it is in all https://github.com/astronomer/astro-sdk/pull/1613/files#diff-3b96a3a64af9191110f3f030c19240c46d17eb4865ec44bcab6bb32acc52ce5bR47 ??

Reply:
@kaxil It's added to __all__. I'm resolving this conversation.